### PR TITLE
CORTX-29092:Fix for logrotation config file.

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
Signed-off-by: Shriya Deshmukh <shriya.deshmukh@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by cdeshmukh — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
looks good

File: src/setup/rgw.py [hunk 1]
```diff
@@ -584,6 +587,8 @@ def _logrotate_generic(conf: MappedConf):
             content = content.replace('TMP_LOG_PATH', log_file_path)
             with open(LOGROTATE_CONF, 'w') as f:
                 f.write(content)
+                f.close()
```
Comment (rahul-tripathi-git) [inline src/setup/rgw.py:590]: Why did we closed the file descriptor explicitly?

Comment (shriya-deshmukh) [reply src/setup/rgw.py:590]: Python closes a file automatically, but It is a good practice to close an opened file.

Comment (gargpalak048) [reply src/setup/rgw.py:590]: with open statement works as a context manager in Python.
No need to explicitly close the file.

Comment (shriya-deshmukh) [reply src/setup/rgw.py]: Removed.

**Review by gargpalak048 — APPROVED** [review]

**Pull request merged**